config/configmap/atlas-dbaas-registry.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mongodb-atlas-registration
+  labels:
+    related-to: dbaas-operator
+    type: dbaas-provider-registration
+data:
+  provider: MongoDB Atlas
+  inventory_kind: MongoDBAtlasInventory
+  connection_kind: MongoDBAtlasConnection
+  credentials_fields: |
+    field1.name: orgId
+    field1.masked: false
```

Comment:
This is already out of date by now! Though I don't know if a PR has been made to the dbaas operator yet

Reply:
Updated the configmap based on the API design doc as of June 16, 2021.